components/centraldashboard/public/components/manage-users-view_test.js

```diff
@@ -1,113 +1,113 @@
 /* eslint-disable max-len */
-// /* eslint-disable max-len */
-// import '@polymer/test-fixture/test-fixture';
-// import 'jasmine-ajax';
-// import {mockIronAjax, yieldForRequests} from '../ajax_test_helper';
-// import {flush} from '@polymer/polymer/lib/utils/flush.js';
-
-// import './dashboard-view';
-// import {languages} from '../assets/i18n/languages.json';
-
-// const FIXTURE_ID = 'manage-users-view-fixture';
```

Comment:
Wow, why did we have this all commented out? Whatever, not like we used this component on the zone

Reply:
I mean if the tests won`t pass (once we are at that ticket) we might also comment it out